scripts/laurian.js

```diff
@@ -40,4 +43,4 @@ $(function() {
 		$("#question").show();
 		$("#answer").hide();
 	});
-});
\ No newline at end of file
+});
```

Comment:
Please break this into a separate whitespace commit

Reply:
WHAT!?

Reply:
IT WAS LIKE THAT WHEN I FOUND IT.